tests/unit/test_env.cpp

```diff
@@ -83,3 +83,20 @@ TEST(convert_array_to_env, MultiEnv) {
 
   destroy_env_list(env_list);
 }
+
+TEST(return_entire_path, lsPath) {
+  char *tmp = getenv("PATH");
+  char **envp = (char **)malloc(sizeof(char *) * 2);
+  envp[0] = ft_strjoin("PATH=", (const char *)(tmp + strlen("PATH: ")));
```

Comment:
```suggestion
  envp[0] = ft_strjoin("PATH=", (const char *)(tmp)));
```

I think getenv return the pointer that points the start of path string.

e.g. on my environment
`printf("%s\n", getenv("PATH"));`
=> `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin`